### PR TITLE
4.0.x fix sdk 33 issue

### PR DESCRIPTION
**Pull request by ghost**
### Platforms affected

Android 

### What does this PR do?

When building against 33 you get this issue:

WindowsTerminal_0lodHODqUd

This removes two deprecated and now removed calls of `settings.setAppCachePath(databasePath)` and `settings.setAppCacheEnabled(true)`. This is removed on upstream since it is not supported in SDK 33 anymore. 

### What testing has been done on this change?

The app now builds:
- Run against SDK 24 Android 7 emulator. The inappbrowser loads normal:
- https://github.com/j5int/cordova-plugin-inappbrowser/assets/25819871/c649c5ed-90ba-4264-804f-0954e0df8863
- Run against SDK 34 Android 14 emulator. The inappbrowser loads normal:
- https://github.com/j5int/cordova-plugin-inappbrowser/assets/25819871/7640e730-f278-41f0-ba2c-3c6cf0e1d6ab

### CODE REVIEW

File: src/android/InAppBrowser.java [hunk 1]
```diff
@@ -1046,8 +1046,6 @@ public void postMessage(String data) {
                     String databasePath = cordova.getActivity().getApplicationContext().getDir("inAppBrowserDB", Context.MODE_PRIVATE).getPath();
                     settings.setDatabasePath(databasePath);
                     settings.setDatabaseEnabled(true);
-                    settings.setAppCachePath(databasePath);
```
Comment (stevehof) [inline src/android/InAppBrowser.java:1049]: What about previous versions of Android? This should have a if 33+ etc etc etc

Comment (ghost) [reply src/android/InAppBrowser.java:1049]: It doesn't even allow me to compile. 
studio64_hjGHAqrqYT

Comment (ghost) [reply src/android/InAppBrowser.java:1049]: Plus it has been removed upstream

Comment (stevehof) [reply src/android/InAppBrowser.java:1049]: Ok cool - can we just test this on an early version of Android Simulator? whats our lowest testing point?

Comment (ghost) [reply src/android/InAppBrowser.java:1049]: Android-12 is making SDK 24 (Android 7) the minimum. Will setup a android 7 emulator and test our app. Will revert back

Comment (ghost) [reply src/android/InAppBrowser.java:1049]: @stevehof  Tested and I added some videos for android 7 and android 14 emulator. The inappbrowser loads normally and I can interact with it.

**Pull request merged**